css/styles.css

```diff
@@ -6,45 +6,56 @@
   box-sizing: border-box;
 }
 
+body {
+  background-color: lightslategrey;
+  margin: 0;
+  padding: 0;
+}
+
 .header {
-  border-radius: 0px 0px 10px 10px;
+  border-radius: 10px 10px 10px 10px;
```

Comment:
you could use a single 10px here

Reply:
-> border-radius: 10px; 
alright will try that!